ckan/cli/cli.py

```diff
@@ -31,19 +32,33 @@
 log = logging.getLogger(__name__)
 
 
+class IClick(click.Group):
+    def get_command(self, ctx, name):
+        cmd = self.commands.get(name)
+        if not cmd:
+            ckan.invoke(ctx)
```

Comment:
and here `self.invoke` is better, as it won't restrict us in future, when new entry points may be added to CLI interface